io.github.torbuntu.Leikr.json

```diff
@@ -0,0 +1,42 @@
+{
+    "id": "io.github.torbuntu.Leikr",
+    "runtime": "org.freedesktop.Platform",
+    "runtime-version": "20.08",
+    "command": "/app/Leikr/Leikr",
+    "sdk": "org.freedesktop.Sdk",
+    "sdk-extensions": ["org.freedesktop.Sdk.Extension.openjdk"],
+    "finish-args": [
+        "--env=PATH=/app/jre/bin:/usr/bin",
+        "--share=ipc",
+        "--socket=x11",
+        "--socket=wayland",
```

Comment:
Does it run under wayland?

Reply:
Yes

Reply:
Updated to use `--socket=fallback-x11` instead of just `x11`

Reply:
Reverted back to `x11` because setting fallback-x11 caused it to break on wayland.